reducer/compile/row.go

```diff
@@ -1,12 +1,16 @@
 package compile
 
 import (
+	"errors"
+
 	"github.com/brimsec/zq/reducer"
 	"github.com/brimsec/zq/zcode"
 	"github.com/brimsec/zq/zng"
 	"github.com/brimsec/zq/zng/resolver"
 )
 
+var ErrDecompose = errors.New("reducer row doesn't decompose")
```

Comment:
Does adding this to the package's public interface meet a need or yield a benefit? If not, just do `return errors.New("reducer row doesn't decompose")` below.